miniapp/miniapp_reduction_to_band.cpp

```diff
@@ -97,6 +97,8 @@ struct Options
 struct reductionToBandMiniapp {
   template <dlaf::Backend backend, typename T>
   static void run(const Options& opts) {
+    using pika::this_thread::experimental::sync_wait;
+
```

Comment:
```suggestion
```

Reply:
Good catch, thanks! Fixed in https://github.com/eth-cscs/DLA-Future/pull/1112/commits/a49c151f29c8e52fe7b7c75859f2afdef2145943.